.github/workflows/update-changelog.yml

```
@@ -91,3 +91,81 @@ jobs:
           delete-branch: true
           add-paths: "CHANGELOG.md"
           sign-commits: true
+
+  update-release:
+    needs: tag-check
+    if: needs.tag-check.outputs.is_stable == 'true'
```

actionlint (1.7.12)

[error] property "is_stable" is not defined in object type {}

(expression)